ui/components/SettingsComponents/GrafanaComponent.js

```diff
@@ -0,0 +1,32 @@
+import React, {useState} from 'react'
```

Comment:
Its a good decision that you've moved this into `SettingsComponents` directory 👍